core/src/memory.rs

```diff
@@ -348,4 +417,19 @@ mod tests {
 
         assert!(gear_pages.eq(&expectation));
     }
+
+    #[test]
+    fn page_buf() {
```

Comment:
What is this test for?

Reply:
To cover page can be crated from vec and debug print works